user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/OnInitInputArgs.md

```diff
@@ -9,10 +9,11 @@ uid: GQI_OnInitInputArgs
 - Namespace: `Skyline.DataMiner.Analytics.GenericInterface`
 - Assembly: `SLAnalyticsTypes.dll`
 
-Contains functionality and general query information for user-definable data sources. An instance is provided through the `OnInit` life cycle method of the [IGQIOnInit building block](xref:GQI_IGQIOnInit).
+Contains functionality and general query information for ad hod data sources and custom operators. An instance is provided through the `OnInit` life cycle method of the [IGQIOnInit building block](xref:GQI_IGQIOnInit).
 
 ## Properties
 
 | Property | Type | Description |
 | -------- | ---- | ----------- |
 | DMS | [GQIDMS](xref:GQI_GQIDMS) | Object that can be used to interact with the DMS. |
+| Logger | [IGQILogger](xref:GQI_IGQILogger) | Object that can be used to log messages and exceptions. |
```

Comment:
Need to mention that this in only available from a specific DataMiner version onwards.